tests/ra_aid/tools/test_reflection.py

```
@@ -0,0 +1,62 @@
+    """A function with type hints and default value.
+    
+    Args:
+        a: An integer parameter
+        b: A string parameter with default
+        
+    Returns:
+        bool: Always returns True
+    """
+    return True
+
+def complex_func(pos1, pos2, *args, kw1="default", **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `complex_func`

Add return type annotation: `None`

(ANN201)